method-info/totals-and-components/totals-and-components.md

```diff
@@ -0,0 +1,143 @@
+
+```python
+# To import this method we are first navigating to the
+# sml_small/editing/totals_and_components directory and
+# the totals_and_components python file
+# to import the totals_and_components function.
+from sml_small.editing.totals_and_components.totals_and_components import totals_and_components
+
+# We can pass our data into the method and save the output to the
+# results variable.
+result = totals_and_components(
+    identifier="1",
+    total=1689,
+    components=
+    [
+        (632),
+        (732),
+        (101),
+        (165)
+    ],
+    amend_total=False,
+    predictive=1689,
+    precision=10,
+    auxiliary=None,
+    absolute_difference_threshold=28,
+    percentage_difference_threshold=0.1
+)
+
+# The output will be returned as an object.
+# You will need to destructure the object to extract the values and
+# one way of doing this is using the built-in function vars().
+# vars() is used to return the __dict__attribute for the specified
+# module, class, instance or any other object with a
+# __dict__attribute
+print(vars(result))
+```
+
+Running this command will then give you the results from the totals and components area.
+
+### Example Output
+
+The output data is determined by the TCC marker. Some values would be returned as None if they are not calculated.
+The output is as follows:
+
+| identifier | absolute_difference | lower_percentage_threshold | upper_percentage_threshold | final_total | final_components | tcc_marker |
+| --- | --- | --- | --- | --- | --- | --- |
+| 1 | 59 | 1467 | 1793 | 1689 | [654.8760735,758.4957055,104.6558282,170.9723927] | "C" | <!-- Components have been corrected  -->
+| 2 | 11 | None | None | 11 | [7,0,2,2] | "T" | <!-- Total value has been corrected -->
+| 3 | None | None | None | 11 | [0,0,0,0] | "S" |  <!-- Method has stopped and no outputs returned -->
+| 4 | None | 9729.9 | 11892.1 | 10811 | [9201,866,632,112] | "N" | <!-- No correction has been applied -->
+| 5 | None | 9729.9 | 11892.1 | 12492 | [9201,866,632,112] | "M" | <!-- Manual editing is required -->
+
+The breakdown of the received outputs are as follows:
+
+* Unique Identifier – Any e.g. Business Reporting Unit
+* Absolute difference - the absolute difference between the predictive value and the sum of the original components
+* Lower percentage threshold - the lower threshold calculated for the percentage range
+* Higher percentage threshold - the higher threshold calculated for the percentage range
+* Final total - the final total will be corrected if applicable or will remain as the original if not
+* Final component - the final components will be corrected if applicable, if not it will remain as original components
+
+### Additional Examples
+
+For more examples of using this method with CSV files or by implementing the pandas package see the totals_and_components source code[documentation page](https://github.com/ONSdigital/sml-python-small/blob/main/sml_small/editing/totals_and_components).
```

Comment:
After "source code", we should have a space before we put the link =)

Reply:
done